ui/app/templates/components/topo-viz/node.hbs

```diff
@@ -6,7 +6,7 @@
       {{else if (not @node.node.isEligible)}}
         <span data-test-status-icon class="tooltip" aria-label="Client is ineligible">{{x-icon "lock-closed" class="is-warning"}}</span>
       {{/if}}
-      <strong>{{@node.node.name}}</strong>
+      <strong><LinkTo @route="clients.client" @model={{@node.node.id}}>{{@node.node.name}}</LinkTo></strong>
```

Comment:
This causes a break in our accessibility testing, however. This might be an issue where our test suite doesn't have a property of `node.node.id` so I'll need to patch the test.